godel_process_execution/src/keyence_process_service.cpp

```diff
@@ -64,65 +57,14 @@ bool godel_process_execution::KeyenceProcessService::executionCallback(godel_msg
 
 bool godel_process_execution::KeyenceProcessService::executeProcess(godel_msgs::KeyenceProcessExecution::Request& req)
 {
-  // Check for keyence existence
-  if (!keyence_client_.exists())
-  {
-    ROS_ERROR_STREAM("Keyence ROS server is not available on service " << keyence_client_.getService());
-    return false;
-  }
-
-  godel_msgs::TrajectoryExecution srv_approach;
-  srv_approach.request.wait_for_execution = true;
-  srv_approach.request.trajectory = req.trajectory_approach;
-
-  godel_msgs::TrajectoryExecution srv_process;
-  srv_process.request.wait_for_execution = true;
-  srv_process.request.trajectory = req.trajectory_process;
-
-  godel_msgs::TrajectoryExecution srv_depart;
-  srv_depart.request.wait_for_execution = true;
-  srv_depart.request.trajectory = req.trajectory_depart;
-
-  if (!real_client_.call(srv_approach))
-  {
-    ROS_ERROR("Execution client unavailable or unable to execute approach trajectory.");
-    return false;
-  }
-
-  keyence_driver::ChangeProgram keyence_srv;
-  keyence_srv.request.program_no = KEYENCE_PROGRAM_LASER_ON;
-
-  if (!keyence_client_.call(keyence_srv))
-  {
-    ROS_ERROR_STREAM("Unable to activate keyence (program " << KEYENCE_PROGRAM_LASER_ON << ").");
-     return false;
-  }
-
-  if (!real_client_.call(srv_process))
-  {
-    ROS_ERROR("Execution client unavailable or unable to execute process trajectory.");
-    return false;
-  }
-
-  // Turn keyence off
-  keyence_srv.request.program_no = KEYENCE_PROGRAM_LASER_OFF;
-  if (!keyence_client_.call(keyence_srv))
-  {
-    ROS_ERROR_STREAM("Unable to de-activate keyence (program " << KEYENCE_PROGRAM_LASER_OFF << ").");
-    return false;
-  }
-
-  if (!real_client_.call(srv_depart))
-  {
-    ROS_ERROR("Execution client unavailable or unable to execute departure trajectory.");
-    return false;
-  }
-
-  return true;
```

Comment:
It's likely that this code will have to be rewritten once we reinstate the keyence dependencies, instead of removing this much code could we use `#if` processor directives until the dependency problem is resolved?  I'm open to suggestions here though.